route/build/src/ncio_utils.f90

```diff
@@ -25,8 +25,10 @@ MODULE ncio_utils
   module procedure get_iscalar
   module procedure get_dscalar
   module procedure get_ivec
+  module procedure get_svec
```

Comment:
I think these are maybe for different types?: integer, single-precision, and double-precision? Might be good to have a one line comment about these to say what they are for...

Reply:
yes, s = single precision, d=double precision, i=integer.  I agree this is not clear, but I will leave them as is for this PR.  Subsequent PR (pio_utils.f90 refactoring) will remove all of these procedures.